src/azul/health.py

```diff
@@ -0,0 +1,47 @@
+import boto3
+
+from botocore.exceptions import ClientError
+
+from azul import config
+
+
+def get_elasticsearch_health():
+    is_es_reachable = config.es_connection().ping()
+
+    result = {
+        'status': 'UP' if is_es_reachable else 'DOWN',
+        'domain': config.es_domain
+    }
+    if not is_es_reachable:
+        result['message'] = 'Unable to reach the host'
+    return result
+
+
+def get_queue_health():
+    notify_queue_name = "notifyQueue"
+    fail_queue_name = "failQueue"
+
+    return {
+        notify_queue_name: __get_queue_health("azul-notify-{config.deployment_stage}".format(config=config)),
+        fail_queue_name: __get_queue_health("azul-fail-{config.deployment_stage}".format(config=config))
+    }
+
+
+def __get_queue_health(queue_name):
+    try:
+        sqs = boto3.resource('sqs')
```

Comment:
Making this a nested function would allow you to share the SQS resource between invocations and eliminate the need for the `__` "protected" convention. See my patch.